corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -449,6 +454,7 @@ def _get_base_context(self):
 
     def _get_column_context(self, column):
         from corehq.apps.app_manager.detail_screen import get_column_generator
+        default_lang = self.app.default_language if not self.build_profile_id else self.app.build_profiles[self.build_profile_id].langs[0]
```

Comment:
nitpick: long line, consider wrapping. I think this is longer than our style recommendation/warning threshold. Are you ignoring @dimagimon in this PR?